packages/material-ui-codemod/README.md

```diff
@@ -31,7 +51,7 @@ A generic codemod to rename any component prop.
 ```
 
 ```sh
-npx jscodeshift --extensions js,ts,jsx,tsx --parser tsx -t node_modules/@material-ui/codemod/node/v5.0.0/component-rename-prop.js --component=Grid --from=prop --to=newProp <path>
+npx @material-ui/codemod v5.0.0/component-rename-prop <path> -- --component=Grid --from=prop --to=newProp
```

Comment:
Should we document?

```suggestion
npx @material-ui/codemod@next v5.0.0/component-rename-prop <path> -- --component=Grid --from=prop --to=newProp
```

Reply:
done: [`9a27f9e` (#26941)](https://github.com/mui-org/material-ui/pull/26941/commits/9a27f9e2e54364698cda3faa3a595569df678e7b)